.goreleaser.develop.yaml

```diff
@@ -0,0 +1,199 @@
+## goreleaser <1.14.0
+project_name: chainlink
+
+env:
+  - ZIG_EXEC={{ if index .Env "ZIG_EXEC"  }}{{ .Env.ZIG_EXEC }}{{ else }}zig{{ end }}
+  - IMAGE_PREFIX={{ if index .Env "IMAGE_PREFIX"  }}{{ .Env.IMAGE_PREFIX }}{{ else }}localhost:5001{{ end }}
+  - IMAGE_LABEL_DESCRIPTION="node of the decentralized oracle network, bridging on and off-chain computation"
+  - IMAGE_LABEL_LICENSES="MIT"
+  - IMAGE_LABEL_SOURCE="https://github.com/smartcontractkit/stage-chainlink"
+
+before:
+  hooks:
+    - go mod tidy
+    - make operator-ui
+    - ./tools/bin/goreleaser_utils before_hook
+
+# See https://goreleaser.com/customization/build/
+builds:
+  - binary: chainlink
+    id: linux-arm64
+    main: ./core
+    hooks:
+      post: ./tools/bin/goreleaser_utils build_post_hook {{ dir .Path }} {{ .Os }} {{ .Arch }}
+    env:
+      - CGO_ENABLED=1
+      - CC=$ZIG_EXEC cc -target aarch64-linux-gnu
+      - CCX=$ZIG_EXEC c++ -target aarch64-linux-gnu
+    flags:
+      - -trimpath
+      - -buildmode=pie
+    ldflags:
+      - -s -w -r=$ORIGIN/libs
+      - -X github.com/smartcontractkit/chainlink/core/static.Version={{ .Version }}
```

Comment:
Does this `.Version` come from the tag and not the `VERSION` file?

Reply:
`.Version` comes from the tag not the `VERSION` file

Reply:
When kicking this off from develop, how would that work? We'd still want some sort of version to be populated but in these cases there's no semver tag, right?

Reply:
by default it will increment the latest tag by the next patch, I can update this to pull from the `VERSION` file or some other mechanism

Reply:
I think that's fine, if it uses the version from the most recent semver tag?

Reply:
Yeah it should. @jmank88 wanted to get rid of the VERSION file too

Reply:
Yea I think we should not rely on the `VERSION` file anymore, going to leave as is...

Reply:
Sounds good as long as we get a correct semantic version passed to the build, even from builds from HEAD of develop.